arangodb/content.md

```diff
@@ -20,36 +18,29 @@ Joins and Transactions are key features for flexible, secure data designs, widel
 
 Furthermore, ArangoDB offers a microservice framework called [Foxx](https://www.arangodb.com/foxx) to build your own Rest API with a few lines of code.
 
-ArangoDB Documentation
-
--	[ArangoDB Documentation](https://www.arangodb.com/documentation)
--	[ArangoDB Tutorials](https://www.arangodb.com/tutorials)
+ArangoDB Documentation - [ArangoDB Documentation](https://www.arangodb.com/documentation) - [ArangoDB Tutorials](https://www.arangodb.com/tutorials)
 
 ## How to use this image
 
-### Start an `ArangoDB` instance
+### Start an ArangoDB instance
 
 In order to start an ArangoDB instance run
 
-```console
-$ docker run -d --name arangodb-instance arangodb
-```
+	unix> docker run -e ARANGO_RANDOM_ROOT_PASSWORD=1 -d --name arangodb-instance -d arangodb/arangodb
 
-Will create and launch the arangodb docker instance as background process. The Identifier of the process is printed - the plain text name will be *arangodb-instance* as you stated above. By default ArangoDB listen on port 8529 for request and the image includes `EXPOST 8529`. If you link an application container it is automatically available in the linked container. See the following examples.
+Will create and launch the arangodb docker instance as background process. The Identifier of the process is printed. By default ArangoDB listen on port 8529 for request and the image includes`EXPOSE 8529`. If you link an application container it is automatically available in the linked container. See the following examples.
 
 In order to get the IP arango listens on run:
 
-```console
-$ docker inspect --format '{{ .NetworkSettings.IPAddress }}' arangodb-instance
-```
+	docker inspect --format '{{ .NetworkSettings.IPAddress }}' <IDENTIFIER>
+
+(where <IDENTIFIER> is the return string of the previous start command)
```

Comment:
This could still be `arangodb-instance`, since that is the `--name` given, but either way is fine.